### DISCUSSION TITLE
Query on Generating Reports for Nested Categories

### DISCUSSION THREAD

**Discussion by expeditus303** [ananthakumaran/paisa#195] [General]
Hello,

I've been using Paisa to manage my finances and I'm currently trying to get more detailed insights into my spending, and I don't know if I'm missing something. Specifically, I want to generate a report for a nested category within my expenses.

Let's say my account structure includes a category like Expenses:Transportation:Auto:Gasoline. I am looking to find out how much I've spent on gasoline for the month. While I can easily check the expenses for the first layer category, such as Expenses:Transportation, I haven't figured out how to view the spend for more specific nested accounts.

Could you guide me on how to generate a report that shows spending on a sub-category such as Expenses:Transportation:Auto:Gasoline? I'm interested in both the total spend in this category for a given time frame and tracking the spending trend over time.

Any tips or guidance would be greatly appreciated.

Thank you for developing such a useful tool and for your support!

Best regards,

Expeditus303

**Comment by ananthakumaran** [comment]
Hierarchical drill down has not yet been implemented, it's been requested by a few users though. As of now, dropping down to ledger cli might be the only option.

**Comment by expeditus303** [comment]
Is hierarchical drill-down on the roadmap, and if so, when might it be available? This feature would be a game changer for those using Paisa and Ledger to check monthly expenses. A chart similar to the yearly cash flow for monthly expenses would resolve the issue and greatly enhance the user experience.
